cardano-node/app/cardano-cli.hs

```diff
@@ -76,21 +84,81 @@ main = do
     renderCliError = show
 
 data CLI = CLI
-  { protocol    :: Protocol
+  { totalConfig :: PartialCardanoConfiguration
```

Comment:
Hmm, `totalConfig` vs.  `Partial*` type? : -)

What is our picture here?  Maybe we want a better name?

Reply:
@deepfire Yep, this was a poor name choice. I'll change it.